src/services/accounts/AccountsBalanceInfoService.ts

```diff
@@ -29,7 +30,7 @@ export class AccountsBalanceInfoService extends AbstractService {
 
 		const at = {
 			hash,
-			height: header.number.toNumber().toString(10),
+			height: header.number.unwrap().toString(10),
```

Comment:
afaiu this isn't anything like the Rust `unwrap()` right? It literally removes the wrapping?

Reply:
haha, yeah this is confusing when `unwrap` is a nasty habit in Rust :)